Products/Zuul/interfaces/template.py

```diff
@@ -148,6 +148,8 @@ class IMinMaxThresholdInfo(IThresholdInfo):
     """
     minval = schema.TextLine(title=_t(u'Minimum Value'), order=6)
     maxval = schema.TextLine(title=u'Maximum Value', order=7)
+    rpnused = schema.Text(title=_t(u'RPN is used'),
```

Comment:
I suggest that "DataPoint RPN(s) in Use" (or just "DataPoint RPN(s)") is a clearer title.